web/markdown/rules.react.js

```diff
@@ -36,3 +36,11 @@ export default function rules() {
     },
   };
 }
+
+// function will contain additional rules for message formatting
+export function advancedRules() {
```

Comment:
And call this `markdownRules`. Also use a single `export { linkRules, markdownRules };` block at the bottom